R/initialization.R

```diff
@@ -94,7 +94,25 @@ get_inits_for_one_chain <- function(stan_data, stdev = 0.01) {
     hosp_wday_effect = to_simplex(abs(
       stats::rnorm(7, 1 / 7, stdev)
     )),
-    infection_feedback = abs(stats::rnorm(1, 500, 20))
+    infection_feedback = abs(stats::rnorm(1, 500, 20)),
+    # Spatial inits
+    log_sigma_generalized = stats::rnorm(1, log(0.01^(n_subpops - 1)), 0.5),
+    log_phi = stats::rnorm(1, log(0.05), 0.1),
+    log_scaling_factor = stats::rnorm(1, log(1), 0.1),
+    non_cent_spatial_dev_ns_mat = matrix(
+      stats::rnorm((n_subpops - 1) * n_weeks,
+        mean = 0,
+        sd = stdev
+      ),
+      (n_subpops - 1),
+      n_weeks
+    ),
+    norm_vec_aux_site = stats::rnorm(n_weeks, 0, stdev)
   )
+  # Initialize the cholesky decomposition matrix if inferring
+  # unstructured correlation matrix
+  if (stan_data$corr_structure_switch == 2) {
```

Comment:
I know I wrote this, but does this throw a warning at the beginning of the inference when the `corr_structure_switch !=2`? I think we should probably make this a `numeric(0)` when `corr_structure_switch !=2`!

Reply:
@kaitejohnson I am not sure what you mean by this, I remember trying to set `init_list$L_Omega = diag(0)` for the case when `corr_structure_switch !=2`.  This is fine with R, but in stan it gave an error.  And yes, there is a warning when `corr_structure_switch !=2`.

Reply:
I'll open a PR onto this branch as a solution!

Reply:
There's some finnicky stuff with R -> stan indexing